src/main/java/org/growify/bank/model/token/Token.java

```diff
@@ -30,4 +30,7 @@ public class Token {
 
     @ManyToOne
     private User user;
-}
+
+    public boolean isTokenExpired() {
```

Comment:
No reason for an explicit getter. Field may have `@lombok.Getter` (which is already implicitly defined by class annotation)

Reply:
Understood. I’ll remove the explicit getters to simplify and follow the standard already used in the class.